inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -428,8 +430,21 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 		}
 	)
 
+	// refetch private repo after login
 	createEffect(() => {
-		if (localStorage?.user?.isLoggedIn) refetchRepoInfo()
+		if (
+			!repo.loading &&
+			repo() === undefined &&
+			githubRepositoryInformation() === undefined &&
+			localStorage?.user?.isLoggedIn
+		) {
+			refetchRepo()
```

Comment:
will this trigger infinite reload when user is logged in and repo cannot be loaded because of other reason?